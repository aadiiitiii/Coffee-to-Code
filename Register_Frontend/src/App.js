/**
  * 1990-2019 Publicis Sapient Corporation. All rights reserved.   
*/

import React from 'react';
import './App.css';
import RegisterButton from './components/RegisterButton';
import { Provider } from 'react-redux';
import store from './store/store';


function App() {
  return (
      <div className="App">
        <RegisterButton />
      </div>
  );
}

export default App;
