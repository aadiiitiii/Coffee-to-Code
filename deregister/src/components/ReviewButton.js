import React, { Component } from 'react'
import { Link } from 'react-router-dom';
import Axios from './Axios';

export default class ReviewButton extends Component {

    render() {
        return (
          <div>
            <br /><br />
            <Link to="/reviews">
              <button className="btn btn-primary"> Reivews</button></Link>
          </div>
        )
      }
    }
