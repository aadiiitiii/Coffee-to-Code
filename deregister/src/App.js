import React from 'react';
import './App.css';
import ReviewButton from "./components/ReviewButton";
function App() {
  return (
   <div className ="App">
     
     <ReviewButton />     
   </div>
   
  );
}

export default App;

