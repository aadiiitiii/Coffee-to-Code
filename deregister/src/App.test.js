import React from 'react';
import { shallow , mount} from 'enzyme';
import App from './App';
import ReviewButton from './components/ReviewButton';

const reviewButton = shallow(<ReviewButton/>);
it('review button renders correctly', () => {
  expect(reviewButton).toMatchSnapshot();
});
