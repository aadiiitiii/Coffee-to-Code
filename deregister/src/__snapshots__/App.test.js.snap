// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `ShallowWrapper {}`;

exports[`review button renders correctly 1`] = `ShallowWrapper {}`;
